source/configuration.txt

```diff
@@ -83,7 +101,18 @@ share the MongoClient across threads.
      - The length of time to keep a ``MongoClient`` available for sharing.
 
        **Default:** ``5000``
-       
+
+``ConfigException``\s
+---------------------
+
+A configuration error throws a ``ConfigException``. Confirm that any of 
+the following methods of configuration that you use are configured 
+properly:
+
+- ``SparkConf``
+- ``ReadConfig`` and ``WriteConfig`` objects
+- Options maps
```

Comment:
issue: I'd like to see a list of "see more" links after the sections, and in this we should link to the API or our own docs so the user doesn't have to hunt for these.